packages/aik-route-config/contextResolution.ts

```diff
@@ -53,7 +53,9 @@ const integration = defineIntegration({
 					},
 				});
 			},
-			'astro:build:setup': ({ pages }) => {
+			'astro:build:setup': ({ pages, target }) => {
+				if (target !== 'server') return;
```

Comment:
Out of curiosity, when is target `"client"`?

Reply:
When building client-side scripts. This hook runs twice